resources/i18n/hi_HI/cura.po

```diff
@@ -0,0 +1,7748 @@
+# Cura
+# Copyright (C) 2021 PBR Research
+# This file is distributed under the same license as the Cura package.
+# Kapil H. Sonone <ksonone8@gmail.com>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Cura 4.10\n"
+"Report-Msgid-Bugs-To: ksonone8@gmail.com\n"
+"POT-Creation-Date: 2021-10-15 11:35+0000\n"
+"PO-Revision-Date: 2021-10-27 1:22+0200\n"
+"Last-Translator: Komal R. Gupta, Kapil H. Sonone <ksonone8@gmail.com>\n"
+"Language-Team: Kapil <www.github.com/ksonone>\n"
+"Language: hi_HI\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n>=2 && n<=4 ? 1 : 2);\n"
+"X-Generator: Poedit 2.4.2\n"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/cura/Machines/Models/DiscoveredPrintersModel.py:83
+#: /home/trin/Gedeeld/Projects/Cura/cura/Machines/Models/IntentCategoryModel.py:110
+#: /home/trin/Gedeeld/Projects/Cura/cura/Machines/Models/QualityManagementModel.py:361
+#: /home/trin/Gedeeld/Projects/Cura/cura/Settings/MachineManager.py:1612
+#: /home/trin/Gedeeld/Projects/Cura/plugins/Toolbox/resources/qml/pages/ToolboxDetailPage.qml:130
+#: /home/trin/Gedeeld/Projects/Cura/plugins/Toolbox/resources/qml/pages/ToolboxDetailPage.qml:171
+msgctxt "@label"
+msgid "Unknown"
+msgstr "अज्ञात"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/cura/Machines/Models/DiscoveredPrintersModel.py:113
+msgctxt "@label"
+msgid "The printer(s) below cannot be connected because they are part of a group"
+msgstr "नीचे दिए गए प्रिंटर को कनेक्ट नहीं किया जा सकता क्योंकि वे एक समूह का हिस्सा है"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/cura/Machines/Models/DiscoveredPrintersModel.py:115
+msgctxt "@label"
+msgid "Available networked printers"
+msgstr "उपलब्ध नेटवर्क प्रिंटर"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/cura/Machines/Models/ExtrudersModel.py:211
+msgctxt "@menuitem"
+msgid "Not overridden"
+msgstr "Nepřepsáno"
```

Comment:
ओवरराइड नहीं किया

Reply:
We'd prefer to communicate in English, if possible, so that everyone can follow along.